.nvmrc

```diff
@@ -1 +1 @@
-v22.16.0
+v24.8.0
```

Comment:
Note that it'll probably be something like v24.11.0 that becomes the LTS that we update to.

Reply:
But we don't know yet, so how can we set the right one? 🤔

Reply:
We can't yet. Just letting you know!